distribution/docs/src/main/resources/content/_reference/_tables/UdpStreamMonitor.adoc

```diff
@@ -33,14 +33,14 @@
 |Max File Size
 |byteCountRolloverCondition
 |Integer
-|Maximum file size (bytes) before rollover. Must be >=1.
+|Maximum file size (bytes) before rollover. Must be >=1. Note that when determining when to finish a chunk, it is looking for certain frame types. When streaming, the video can have fewer of these frames, which can lead to larger chunks.
 |10000000
 |false
 
 |Max Elapsed Time
 |elapsedTimeRolloverCondition
 |Long
-|Maximum elapsed time in milliseconds before rollover. Must be >=1.
+|Maximum elapsed time in milliseconds before rollover. Must be >=1. Note that when determining when to finish a chunk, it is looking for certain frame types. When streaming, the video can have fewer of these frames, which can lead to taking more time to process each chunk.
```

Comment:
Instead of "taking more time to process each chunk", I think it would be more accurate to say something like "the rollover may be delayed beyond this time limit"